grails-app/conf/BootStrap.groovy

```diff
@@ -194,13 +194,13 @@ class BootStrap {
                 name: orderSummary?.order?.name,
                 origin: orderSummary?.order?.origin?.name + (origOrgCode ? " (${origOrgCode})" : ""),
                 destination: orderSummary?.order?.destination?.name + (destOrgCode ? " (${destOrgCode})" : ""),
-                orderedOn: orderSummary?.order?.dateOrdered,
+                dateOrdered: orderSummary?.order?.dateOrdered,
```

Comment:
had to change it, because when fetching it was `orderedOn`, and then react-table when sorting was assigning it as `orderedOn`, but to sort by date we have to use `dateOrdered`, so not to make unnecessary ifs on the frontend side, I decided to change it here, as it is not a big 'game changer'